MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/query/HighlightedLineActivity.java

```diff
@@ -80,26 +80,26 @@ public void onStyleLoaded(@NonNull Style style) {
 
   @Override
   public boolean onMapClick(@NonNull LatLng point) {
-    if (!style.isFullyLoaded()) {
```

Comment:
Should we remove the unused `style` declaration?

Reply:
Don't think there's anything to remove. Right now, `onMapClick()` is 

```java
 @Override
  public boolean onMapClick(@NonNull LatLng point) {
    mapboxMap.getStyle(new Style.OnStyleLoaded() {
      @Override
      public void onStyleLoaded(@NonNull Style style) {
        // Detect whether a linestring was clicked on
        PointF pointf = mapboxMap.getProjection().toScreenLocation(point);
        RectF rectF = new RectF(pointf.x - 10, pointf.y - 10, pointf.x + 10, pointf.y + 10);
        List<Feature> featureList = mapboxMap.queryRenderedFeatures(rectF, "line-layer-id");
        if (featureList.size() > 0) {
          for (Feature feature : featureList) {
            GeoJsonSource source = style.getSourceAs("background-geojson-source-id");
            if (source != null) {
              source.setGeoJson(feature);
              backgroundLineLayer.setProperties(visibility(VISIBLE));
            }
          }
        }

      }
    });
    return true;
  }
```

`style.getSourceAs` is used, which is why all of this is happening in the onStyleLoadedCallback

Reply:
Oh, I see what you mean. The variable declared at the top of the activity. Removed it.